src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +70,73 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+}
+
+class Block{
+	
+	private String blockNo;
+	
+	// Default Constructor
+	public Block(){
+		this.blockNo = "000";
+	}
+	
+	public Block(String blockNo){
+		this.blockNo = blockNo;
+	}
+	public String getBlock(){
+		return blockNo;
+	}
+}
+
+class Street{
+	
+	private String streetName;
+	
+	// Default Constructor
+	public Street(){
+		this.streetName = "default";
```

Comment:
Indentation problem all over the code?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.